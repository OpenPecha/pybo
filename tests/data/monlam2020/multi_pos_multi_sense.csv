edited,favorites,_id,word,result
-,-,83307,ལ་,མིང་ཚིག 1. བོད་ཡིག་གི་གསལ་བྱེད་ལས་སྡེ་བ་བདུན་པའི་ཡི་གེ་གཉིས་པ་སྟེ། འདིའི་ང་རོ་འདོན་ཚུལ་ནི། སྐྱེ་གནས་སོ་དང༌། བྱེད་པ་ལྕེ་རྩེ། སྒྲ་འབྱིན་ཚུལ་ཆེས་ཤིན་ཏུ་ལྷོད་པ། མིང་གཞིའི་རྟགས་ཀྱི་དབྱེ་བའི་མོ་གཤམ་གྱི་ཡི་གེ་ཞིག 2. རྗེས་འཇུག་བཅུའི་དགུ་པ་སྟེ། མིང་གཞིའི་ཡི་གེ་ཀུན་གྱི་རྗེས་སུ་འཇུག  དཔེར་ན། ཀལ། ཁལ། གལ། ངལ། ཅལ། ཆལ། མཇལ། ཉལ། ཏལ། ཐལ། དལ། ནལ། དཔལ། ཕལ། བལ། མལ། བཙལ། ཚལ། འཛོལ། ཝལ། ཞལ། ཟལ། འོལ། ཡལ། རལ། ཤལ། སལ། ཧལ། ཨལ་བཅས་པ་ལྟ་བུ། 3. ༡རྒྱུན་སྤྱོད། རི་འདབས་ནས་ཡར་རི་རྩེ་བར་གྱི་གྱེན་ངོས་སམ། ས་བབ་དམའ་ས་ནས་མཐོ་ས་ལ་ཡར་གྱེན་དུ་གསེག་ཡོད་པའི་ཟུར་ངོས་ཀྱི་མིང། དཔེར་ན། ལ་རེ་ཐུར་རེ་མ་བརྒྱབ་ན། བདེ་མོའི་ཐང་ལ་ཐོན་རྒྱུ་མེད་ལྟ་བུ། ༡ལྡོག་ཚིག ཐུར། 4. ༡ཆོས་ལུགས། གཅོད་བྱེད་ཀྱི་ཡི་གེ་སྟེ། སྐུ་ཐམས་ཅད་ཀྱི་སྣང་བ་སྟོན་པ་དབང་རྫོགས་པ་རང་བྱུང་ཆེན་པོའི་རྒྱུད་ལས། ལ་ནི་གཅོད་བྱེད་ཀྱི་ཡི་གེར་ཤེས་པར་བྱའོ། །ཞེས་རྒྱུད་དེའི་སྒྲའི་སྡོམ་བྱང་བསྟན་པའི་སྐབས་སུ་གསལ། བྱེད་ཚིག 1. ༡རྒྱུན་སྤྱོད། ལས་སུ་བྱ་བ་ལ་འཇུག་པ། ལ་དོན་གྱི་ལ་ཡིག་སྦྱར་བའི་གཞི་དེ་ལ། བྱ་བ་གཞན་ཞིག་བྱས་པའམ་བྱེད་བཞིན་པ་དང་། བྱེད་འགྱུར་གང་རུང་སྟོན་པ་ལ་འཇུག་པ་སྟེ། སྦྱོར་ཚུལ་ནི་རྗེས་འཇུག་ཐམས་ཅད་ཀྱི་རྗེས་སུ་ཁྱད་མེད་དུ་འཇུག་གོ།  དཔེར་ན། ཤར་ཕྱོགས་ལ་འགྲོ། རང་ཁྱིམ་ལ་བསྡད་ལྟ་བུ། 2. ༡རྒྱུན་སྤྱོད། དགོས་ཆེད་ལ་འཇུག་པ། ལ་ཡིག་སྦྱར་བའི་ཚེ་ན། བྱེད་པ་པོ་ཅི་རིགས་ཤིག་གིས་རང་གི་དགོས་པའམ་བྱ་བའི་ཡུལ་གྱི་དགོས་པ་ཞིག་བསྒྲུབ་པའི་ཆེད་དུ་བྱ་བ་ཞིག་བྱེད་པར་སྟོན་པ་སྟེ། སྦྱོར་ཚུལ་ནི་རྗེས་འཇུག་ཐམས་ཅད་ཀྱི་རྗེས་སུ་ཁྱད་མེད་དུ་འཇུག་གོ།  དཔེར་ན། ནད་པ་ལ་སྨན་བྱིན། མེ་ཏོག་ལ་ཆུ་འདྲེན་ལྟ་བུ། 3. ༡རྒྱུན་སྤྱོད། རྟེན་གནས་ལ་འཇུག་པ། ལ་ཡིག་སྦྱར་ཚེ་དངོས་པོ་གང་ཞིག་གང་དུ་ཡོད་པར་སྟོན་པར་བྱེད་པ་སྟེ། སྦྱོར་ཚུལ་ནི་རྗེས་འཇུག་ཐམས་ཅད་ཀྱི་རྗེས་སུ་ཁྱད་མེད་དུ་འཇུག་གོ།  དཔེར་ན། ཐབ་ལ་མེ་ཡོད། ཁང་ཐོག་ལ་དུད་ཁུང་ཡོད་ལྟ་བུ། 4. ༡རྒྱུན་སྤྱོད། དེ་ཉིད་ལ་འཇུག་པ། ལ་དོན་གྱི་ལ་ཡིག་སྦྱར་ས་དེའི་ངོ་བོར་གྲུབ་པའམ་དེ་ཉིད་དུ་ངེས་པར་སྟོན་པ་དོན་གྱི་ཆ་ནས་ཡུལ་དང་བྱ་བ་སོ་སོར་འབྱེད་དུ་མེད་པ་ཞིག་སྟོན་པ་སྟེ། སྦྱོར་ཚུལ་ནི་རྗེས་འཇུག་ཐམས་ཅད་ཀྱི་རྗེས་སུ་ཁྱད་མེད་དུ་འཇུག་གོ།  དཔེར་ན། ཇེ་གསལ་ལ་བཏང། བདེན་པ་ལ་གནས་ལྟ་བུ། 5. ༡རྒྱུན་སྤྱོད། ཚེ་སྐབས་ལ་འཇུག་པ། དུས་དང་སྐབས་སྟོན་པའི་མིང་མཐར་ལ་ཡིག་སྦྱར་ནས། བྱ་བ་གང་ཞིག་གམ་འགྱུར་ལྡོག་གང་ཞིག་དུས་སྐབས་ག་འདྲ་ཞིག་ལ་བྱེད་པའམ་འབྱུང་བ་སྟོན་པ་སྟེ། སྦྱོར་ཚུལ་ནི་རྗེས་འཇུག་ཐམས་ཅད་ཀྱི་རྗེས་སུ་ཁྱད་མེད་དུ་འཇུག་གོ།  དཔེར་ན། ཚེས་བཅོ་ལྔ་ལ་འཁྲུངས། ནམ་གུང་ཙམ་ལ་འབྱོར་ལྟ་བུ། 6. ༡རྒྱུན་སྤྱོད། ཚིག་སྔ་ཕྱི་གཉིས་ཀྱི་མཚམས་སྦྱོར་དུ་འཇུག་པ་སྟེ།  དཔེར་ན། རྩམ་པ་ལྡོད་ལ་མིད། གླུ་ལོངས་ལ་གར་རྩེན་ལྟ་བུ། 7. ༡རྒྱུན་སྤྱོད། དོན་རྐྱེན་གསལ་བྱེད་དུ་འཇུག་པ་ནི།  དཔེར་ན། རྒྱལ་པོའི་ཆད་པ་ལ་བྲོས་ལྟ་བུ། 8. ༡རྒྱུན་སྤྱོད། བྱེད་རྒྱུ་གསལ་བྱེད་དུ་འཇུག་པ།  དཔེར་ན། དངུལ་དཀར་སྟོང་ཕྲག་གཅིག་ལ་བྱམས་པའི་སྐུ་བཞེངས་ལྟ་བུ་སྟེ། དངུལ་དཀར་སྟོང་ཕྲག་གཅིག་གིས་བྱམས་པའི་སྐུ་བཞེངས་ཞེས་པ་དང་དོན་གཅིག་གོ། 9. ༡རྒྱུན་སྤྱོད། བརྗོད་གཞི་ངོས་འཛིན་པའི་དོན་དུ་འཇུག་པ་ནི།  དཔེར་ན། དགའ་བའི་ལུས་ལ་རང་དབང་མེད། སྐྱིད་པའི་སེམས་ལ་དྲན་འཛིན་མེད་ལྟ་བུ། གྲོགས་ཚིག བྱ་ཚིག་མ་འོངས་པའི་མཐའ་ལ་སྦྱར་ན་བྱ་བ་དེ་བྱེད་པ་ལ་ཧ་ཅང་ཉེ་བའི་དོན་སྟོན་ཏེ།  དཔེར་ན། ང་གནས་གཞན་དུ་འགྲོ་ལ་ཡོད་ལྟ་བུ།
