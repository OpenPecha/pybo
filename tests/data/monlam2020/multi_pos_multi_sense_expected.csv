ID,Form,Lemma,MonPOS,MonFeature,MonTag,POS,Feature,Morph,SenseTag,Definition,Example
1,ལ་,ལ་1,མིང་ཚིག,,,,,,བོད་ཡིག,བོད་ཡིག་གི་གསལ་བྱེད་ལས་སྡེ་བ་བདུན་པའི་ཡི་གེ་གཉིས་པ་སྟེ། འདིའི་ང་རོ་འདོན་ཚུལ་ནི། སྐྱེ་གནས་སོ་དང༌། བྱེད་པ་ལྕེ་རྩེ། སྒྲ་འབྱིན་ཚུལ་ཆེས་ཤིན་ཏུ་ལྷོད་པ། མིང་གཞིའི་རྟགས་ཀྱི་དབྱེ་བའི་མོ་གཤམ་གྱི་ཡི་གེ་ཞིག,
2,ལ་,ལ་1,མིང་ཚིག,,,,,,རྗེས་འཇུག,རྗེས་འཇུག་བཅུའི་དགུ་པ་སྟེ། མིང་གཞིའི་ཡི་གེ་ཀུན་གྱི་རྗེས་སུ་འཇུག ,ཀལ། ཁལ། གལ། ངལ། ཅལ། ཆལ། མཇལ། ཉལ། ཏལ། ཐལ། དལ། ནལ། དཔལ། ཕལ། བལ། མལ། བཙལ། ཚལ། འཛོལ། ཝལ། ཞལ། ཟལ། འོལ། ཡལ། རལ། ཤལ། སལ། ཧལ། ཨལ་བཅས་པ་ལྟ་བུ།
3,ལ་,ལ་1,མིང་ཚིག,,"༡རྒྱུན་སྤྱོད།, ༡ལྡོག་ཚིག:ཐུར།",,,,རི་འདབས,རི་འདབས་ནས་ཡར་རི་རྩེ་བར་གྱི་གྱེན་ངོས་སམ། ས་བབ་དམའ་ས་ནས་མཐོ་ས་ལ་ཡར་གྱེན་དུ་གསེག་ཡོད་པའི་ཟུར་ངོས་ཀྱི་མིང། ,ལ་རེ་ཐུར་རེ་མ་བརྒྱབ་ན། བདེ་མོའི་ཐང་ལ་ཐོན་རྒྱུ་མེད་ལྟ་བུ། 
4,ལ་,ལ་1,མིང་ཚིག,,༡ཆོས་ལུགས།,,,,གཅོད་བྱེད,གཅོད་བྱེད་ཀྱི་ཡི་གེ་སྟེ། སྐུ་ཐམས་ཅད་ཀྱི་སྣང་བ་སྟོན་པ་དབང་རྫོགས་པ་རང་བྱུང་ཆེན་པོའི་རྒྱུད་ལས། ལ་ནི་གཅོད་བྱེད་ཀྱི་ཡི་གེར་ཤེས་པར་བྱའོ། །ཞེས་རྒྱུད་དེའི་སྒྲའི་སྡོམ་བྱང་བསྟན་པའི་སྐབས་སུ་གསལ།,
5,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
6,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
7,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
8,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
9,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
10,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
11,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
12,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
13,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
14,ལ་,ལ་2,བྱེད་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
15,ལ་,ལ་3,གྲོགས་ཚིག,,༡རྒྱུན་སྤྱོད།,,,,,,
